CITATION.cff

```diff
@@ -77,6 +77,18 @@ references:
     authors:
       - given-names: Marshall D.
         family-names: Perrin
+      - given-names: Anand
+        family-names: Sivaramakrishnan
+      - given-names: Charles-Philippe
+        family-names: Lajoie
+      - given-names: Erin
+        family-names: Elliott
+      - given-names: Laurent
+        family-names: Pueyo
+      - given-names: Swara
+        family-names: Ravindranath
+      - given-names: Loïc
+        family-names: Albert
     title: "Simulating point spread functions for the James Webb Space Telescope with WebbPSF"
```

Comment:
This should be "Updated point spread function simulations for JWST with WebbPSF"